### ISSUE TITLE
add seed option for all the simulation/random draw functions

### ISSUE THREAD

**Issue by carlbfrederick** [jknowles/merTools#23]
It looks like at least `FEsim` does not respect seed setting outside the function UNLESS it was set immediately before the call to FEsim. Is this how all R functions work?  

If not, we should probably go through all the functions and add options + ensure that the functions get a random seed set outside the functions.

If so, it is good to know :)

**Comment by jknowles** [comment]
OK, great. TIL that R does not support setting a seed for an entire session: http://stackoverflow.com/questions/20624698/fixing-set-seed-for-an-entire-session

So we can either set it as an option and put a code snippet in our functions that gets the option -- which I like -- or make it a user parameter that is optional. Either one seems fine with me.

**Comment by carlbfrederick** [comment]
`lme4::bootMer` has an optional argument like this:

```
  if (!is.null(seed)) 
    set.seed(seed)
  else if (!exists(".Random.seed", envir = .GlobalEnv)) 
    runif(1)
```

the call to `runif` causes R to set the seed

So far, the list of candidate functions for this enhancement are:

  - `predictInterval`
  - `draw.random` / `randomObs`
  - `subBoot`

Let me know if there are others

**Comment by jknowles** [comment]
- `REsim`
- `FEsim`

**Comment by jknowles** [comment]
Closed by #36
